python/cuml/common/input_utils.py

```diff
@@ -200,6 +201,7 @@ def is_array_like(X):
     return determine_array_type(X) is not None
 
 
+@nvtx.annotate(message="cuml.common.input_to_cuml_array")
```

Comment:
```suggestion
@nvtx.annotate(message="cuml.common.input_utils.input_to_cuml_array")
```
I think adding the full path might be good so people know where to look for the function quickly in source code, what do you think?